oxcaml/tests/backend/llvmize/dune.inc

```diff
@@ -154,3 +158,38 @@
  (deps array_rev.output array_rev.output.corrected)
  (action
   (diff array_rev.output array_rev.output.corrected)))
+
+(rule
+ (enabled_if (and (= %{context_name} "main") (= %{architecture} "amd64")))
+ (targets float_ops.output.exe float_ops_ir.output.corrected)
+ (deps  float_ops_main.ml float_ops.ml)
+ (action
+  (progn
+   
+   (run %{bin:ocamlopt.opt} float_ops.ml -c -g -O3 -opaque -llvm-backend -llvm-path clang -keep-llvmir -dno-asm-comments)
+   (run %{bin:ocamlopt.opt} float_ops_main.ml -c -g -O3 -opaque)
+   (run %{bin:ocamlopt.opt}  float_ops.cmx float_ops_main.cmx -opaque -o float_ops.output.exe)
```

Comment:
Consider adding more options here to help debugging, such as "-S -dump-into-file -dcmm -dcfg -dlinear".